docs/FAQ.md

```diff
@@ -16,7 +16,7 @@ architectures is currently only available from the
 
 For other architectures, we recommend that you try out the
 [Splunk Operator for Kubernetes](https://splunk.github.io/splunk-operator/).
-This open source project is currently a “beta,” and we welcome any
+This open source project is currently a “beta” and we welcome any
```

Comment:
We should change this "This open-source project is currently a “beta” to "This open-source project and we welcome any"

Reply:
Good catch Kriti, but we will update this part during the GA release